Cubical/Algebra/CommRing/FGIdeal.agda

```diff
@@ -265,79 +266,80 @@ module GeneratingExponents (R' : CommRing ℓ) (f g : fst R') (n : ℕ) where
   R = fst R'
   ⟨_⟩ : {n : ℕ} → FinVec R n → CommIdeal
   ⟨ V ⟩ = ⟨ V ⟩[ R' ]
-  fgVec : FinVec R 2
-  fgVec zero = f
-  fgVec (suc zero) = g
-  ⟨f,g⟩ = ⟨ fgVec ⟩
+  _ⁿ : {m : ℕ} → FinVec R m → FinVec R m
+  U ⁿ = λ i → U i ^ n
 
-  fⁿgⁿVec : FinVec R 2
-  fⁿgⁿVec zero = f ^ n
-  fⁿgⁿVec (suc zero) = g ^ n
 
-  ⟨fⁿ,gⁿ⟩ = ⟨ fⁿgⁿVec ⟩
-
- lemma : 1r ∈ ⟨f,g⟩ → 1r ∈ ⟨fⁿ,gⁿ⟩
- lemma = elim (λ _ →  isPropPropTrunc) Σlemma
+ lemma : (m : ℕ) (α U : FinVec R (ℕsuc m))
+       → (linearCombination R' α U) ^ ((ℕsuc m) ·ℕ n) ∈ ⟨ U ⁿ ⟩
+ lemma ℕzero α U = ∣ α ⁿ , path ∣
+  where
+  path : (α zero · U zero + 0r) ^ (n +ℕ 0) ≡ α zero ^ n · U zero ^ n + 0r
+  path = (α zero · U zero + 0r) ^ (n +ℕ 0) ≡⟨ cong (_^ (n +ℕ 0)) (+Rid _) ⟩
+         (α zero · U zero) ^ (n +ℕ 0)      ≡⟨ cong ((α zero · U zero) ^_) (+-zero n) ⟩
+         (α zero · U zero) ^ n             ≡⟨ ^-ldist-· _ _ n ⟩
+         α zero ^ n · U zero ^ n           ≡⟨ sym (+Rid _) ⟩
+         α zero ^ n · U zero ^ n + 0r ∎
+
+ lemma (ℕsuc m) α U = subst-∈ ⟨ U ⁿ ⟩ (sym (BinomialThm (n +ℕ (ℕsuc m) ·ℕ n) x y)) ∑Binomial∈⟨Uⁿ⟩
   where
-  Σlemma : Σ[ α ∈ FinVec R 2 ] (1r ≡ linearCombination R' α fgVec) → 1r ∈ ⟨fⁿ,gⁿ⟩
-  Σlemma (α , p) = subst-∈ ⟨fⁿ,gⁿ⟩ path ∑Binomial∈⟨fⁿ,gⁿ⟩
+  x = α zero · U zero
+  y = linearCombination R' (α ∘ suc) (U ∘ suc)
+  -- x + y = linearCombination R' α U
```

Comment:
Why is this comment here?

Reply:
the proof relies on those two being the same definitionally, but I removed anyways